packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -955,7 +955,7 @@ export class Project extends ProjectBase {
     }));
 
     const policy = new iam.Policy(this, 'PolicyDocument', {
-      policyName: 'CodeBuildEC2Policy',
+      policyName: this.node.uniqueId + ' CodeBuildEC2Policy',
```

Comment:
Are you sure you want a space here?

Reply:
It's okay if there's no space.

Reply:
No. You misunderstood me. There is a space there that should not be. I'm don't think you can have spaces in policy names.

Reply:
Yes,
I understood you mean that space is in policy name.
Should I modify it like this? :
`this.node.uniqueId + 'CodeBuildEC2Policy'`

Reply:
Yes